content/zh-cn/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -1,801 +1,798 @@
+内存管理器也会为 Pods 应用 `cgroups` 设置（即 `cpuset.mems`）。
+与 Pod 准入和部署流程相关的完整流程图在[Memory Manager KEP: Design Overview][4]，
+下面也有说明。
+
+<!--
+![Memory Manager in the pod admission and deployment process](/images/docs/memory-manager-diagram.svg)
+-->
+![Pod 准入与部署流程中的内存管理器](/images/docs/memory-manager-diagram.svg)
+
+<!--
+During this process, the Memory Manager updates its internal counters stored in
+[Node Map and Memory Maps][2] to manage guaranteed memory allocation. 
+
+The Memory Manager updates the Node Map during the startup and runtime as follows.
+-->
+在这个过程中，内存管理器会更新其内部存储于[节点映射和内存映射][2]中的计数器，
+从而管理有保障的内存分配。
+
+内存管理器在启动和运行期间按下述逻辑更新节点映射（Node Map）。
+
+<!--
+### Startup
+-->
+### 启动  {#startup}
+
+<!--
+This occurs once a node administrator employs `--reserved-memory` (section
+[Reserved memory flag](#reserved-memory-flag)).
+In this case, the Node Map becomes updated to reflect this reservation as illustrated in
+[Memory Manager KEP: Memory Maps at start-up (with examples)][5].
+
+The administrator must provide `--reserved-memory` flag when `Static` policy is configured.
+-->
+当节点管理员应用 `--reserved-memory` [预留内存标志](#reserved-memory-flag)时执行此逻辑。
+这时，节点映射会被更新以反映内存的预留，如
+[Memory Manager KEP: Memory Maps at start-up (with examples)][5]
+所说明。
+
+当配置了 `Static` 策略时，管理员必须提供 `--reserved-memory` 标志设置。
+
+<!--
+### Runtime
+-->
+### 运行时  {#runtime} 
+
+<!--
+Reference [Memory Manager KEP: Memory Maps at runtime (with examples)][6]
+illustrates how a successful pod deployment affects the Node Map, and it also relates to
+how potential Out-of-Memory (OOM) situations are handled further by Kubernetes or operating system.
+-->
+参考文献 [Memory Manager KEP: Memory Maps at runtime (with examples)][6]
+中说明了成功的 Pod 部署是如何影响节点映射的，该文档也解释了可能发生的内存不足
+（Out-of-memory，OOM）情况是如何进一步被 Kubernetes 或操作系统处理的。
+
+<!--
+Important topic in the context of Memory Manager operation is the management of NUMA groups.
+Each time pod's memory request is in excess of single NUMA node capacity, the Memory Manager
+attempts to create a group that comprises several NUMA nodes and features extend memory capacity.
+The problem has been solved as elaborated in
+[Memory Manager KEP: How to enable the guaranteed memory allocation over many NUMA nodes?][3].
+Also, reference [Memory Manager KEP: Simulation - how the Memory Manager works? (by examples)][1]
+illustrates how the management of groups occurs. 
+-->
+在内存管理器运作的语境中，一个重要的话题是对 NUMA 分组的管理。
+每当 Pod 的内存请求超出单个 NUMA 节点容量时，内存管理器会尝试创建一个包含多个
+NUMA 节点的分组，从而扩展内存容量。解决这个问题的详细描述在文档
+[Memory Manager KEP: How to enable the guaranteed memory allocation over many NUMA nodes?][3]
+中。同时，关于 NUMA 分组是如何管理的，你还可以参考文档
+[Memory Manager KEP: Simulation - how the Memory Manager works? (by examples)][1]。
+
+<!--
+## Memory Manager configuration
+-->
+## 内存管理器配置   {#memory-manager-configuration}
+
+<!--
+Other Managers should be first pre-configured. Next, the Memory Manger feature should be enabled
+and be run with `Static` policy (section [Static policy](#static-policy)).
```

Comment:
```suggestion
and be run with `Static` policy (section [Static policy](#policy-static)).
```

Reply:
It's updated in English version.

https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/administer-cluster/memory-manager.md?plain=1#L96